.travis.yml

```diff
@@ -132,6 +132,12 @@ matrix:
     env: BEAKER_set="ubuntu-1804" BEAKER_PUPPET_COLLECTION=puppet6
     bundler_args:
     script: bundle exec rake beaker
+  - rvm: 2.5.1
```

Comment:
This should also test against puppet5 like the other platforms

Reply:
The problem is that puppet 5 is not supported in Ubuntu 20.04, so it can't be checked.

Reply:
Makes sense. Could you please place a comment above this that explains this.

Reply:
@amateo This file still needs the comment regarding puppet5